frontend/Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM node:10-slim as node
 
-# install build toolchain (for node-gyp)
+# install build toolchain
 RUN apt-get update \
-    && apt-get install --no-install-recommends -y dumb-init python build-essential \
+    && apt-get install --no-install-recommends -y dumb-init \
```

Comment:
I have dealt with the `node-gyp` errors on this PR so we no longer need python and the C++ compiler on our docker images to rebuild it.

Original GH issue: https://github.com/fsevents/fsevents/issues/301